scripts/check_compatibility_careamics.py

```diff
@@ -0,0 +1,127 @@
+from bioimageio.spec import load_model_description
+from bioimageio.spec.generic.v0_2 import AttachmentsDescr
+
+from .script_utils import CompatibilityReportDict, check_tool_compatibility
+
+
+def check_compatibility_careamics_impl(
+    rdf_url: str,
+    sha256: str,
+) -> CompatibilityReportDict:
+    """Create a `CompatibilityReport` for a resource description.
+
+    Args:
+        rdf_url: URL to the rdf.yaml file
+        sha256: SHA-256 value of **rdf_url** content
+    """
+    model_desc = load_model_description(rdf_url)
+    if isinstance(model_desc.attachments, AttachmentsDescr):
+        attachment_file_names = [
+            Path(file.path).name
+            for file in model_desc.attachments.files
+            if file.path is not None
+        ]
+    elif isinstance(model_desc.attachments, list):
+        attachment_file_names = [
+            Path(attachment.source.path).name
+            for attachment in model_desc.attachments
+            if attachment.source.path is not None
+        ]
+    else:
+        # TODO: confirm all types of attachments (type checker still complaining)
+        report = CompatibilityReportDict(
+            status="failed",
+            error=None,
+            details="Could not process attachments.",
+        )
+        return report
+
+    # check type is tagged as CAREamics
+    if ("CAREamics" not in model_desc.tags) and ("careamics" not in model_desc.tags):
+        report = CompatibilityReportDict(
+            status="not-applicable",
+            error=None,
+            details="'Model' resource not tagged with 'CAREamics' or 'careamics'.",
+        )
+    # check config file is present in attachments
+    # TODO: update to careamics.yaml once files have been updated
+    elif not "config.yml" in attachment_file_names:
+        report = CompatibilityReportDict(
+            status="failed",
+            error=None,
+            details="CAREamics config file is not present in attachments.",
+        )
+    # download and test
+    else:
+        try:
+            model, config = load_from_bmz(rdf_url)
+        except (ValueError, pydantic.ValidationError) as e:
+            report = CompatibilityReportDict(
+                status="failed",
+                error="Error: {}".format(e),
+                details=("Could not load CAREamics configuration or model."),
+            )
+            return report
+
+        # no failure mode as config is already a Configuration object
+        careamist = CAREamist(config)
+        # TODO (CAREamics): make a model loading method, why doesn't this exist
+        careamist.model = model
+
+        # get input tensor
+        # TODO: type checker complaining because of difference between v0.4 and v0.5
+        #   test_tensor attribute does not exist for v0.4,
+        #   how can the tensor path be accessed?
+        input_path = model_desc.inputs[0].test_tensor.download().path
+        input_array = np.load(input_path)
```

Comment:
we could let bioimageio.core deal with it and use https://bioimage-io.github.io/core-bioimage-io-python/bioimageio/core/digest_spec.html#get_test_inputs here

Reply:
then you'd also have axis annotations and could transpose to your desired SCZYX order